src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetBuilding.java

```diff
@@ -91,15 +72,15 @@ private static void setToolMode(ItemStack stack, ToolMode mode) {
         stack.setTagCompound(tagCompound);
     }
 
-    public static ToolMode getToolMode(ItemStack stack) {
+    public static com.direwolf20.buildinggadgets.common.tools.BuildingModes getToolMode(ItemStack stack) {
```

Comment:
```suggestion
    public static BuildingModes getToolMode(ItemStack stack) {
```
You already imported the entire `common.tools` package. All instances of this are redundant.

Reply:
You have marked this as resolved, but still have 2 other instances of this in this class (as well as 1 instance in GadgetExchanger).

Reply:
There's still one instance left in this class.